tests/e2e/test_diffusion.py

```diff
@@ -34,6 +34,10 @@
 from tests.utils import ensure_similar_images
 
 
+def _img_open(pth: Path) -> Image.Image:
```

Comment:
Is `pth` intended (short form) or you just wanted to use `path` instead?

Reply:
Hmm, intended, I tend to name paths `pth` (to avoid conflicts with `from os import path` mostly). I can rename it if you prefer.

Reply:
Given that we have `path: ...,` elsewhere I'd say. But no big deal.